package.json

```diff
@@ -28,7 +28,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@types/bcrypt": "^5.0.0",
+    // "@types/bcrypt": "^5.0.0",
```

Comment:
Remove the all comments from package.json, as it's will cause an error when someone tries to do `npm install`.

Reply:
Uninstall will be great